api/helpers/actionHelper.js

```diff
@@ -1,5 +1,5 @@
 function updateStyles(styleUpdates, trace) {
```

Comment:
Can you add comments for functions in this file?,

Reply:
Done